examples/dagger/Notebook.md

```diff
@@ -9,23 +9,22 @@ Showcase the notebook experience to **author, debug, express, and run** Dagger p
 ### Let's use a Dagger function to build the Runme binary
 
 ```sh {"id":"01HZSMYF33TFKMEVRX5P64BNTB","interactive":"true","name":"RUNME_BINARY"}
-dagger --progress=$PROGRESS \
-    call \
-        -m github.com/purpleclay/daggerverse/golang@7e83bccc350fa981e975ac0c8619f92a1b729958 \
-        --src "https://github.com/stateful/runme#main" \
-    build \
-        --arch $(go env GOARCH) \
-        --os $(go env GOOS) \
-    file \
-        --path runme
+$ dagger call \
+  -m github.com/purpleclay/daggerverse/golang \
+    --src 'https://github.com/stateful/runme#main' \
+  build  \
+    --arch $(go env GOARCH) --os $(go env GOOS) \
+  file \
+    --path runme
+
 ```
 
 ### What does the 🐮 cow say?
 
 ```sh {"id":"01J022WD7Z6TM1QQ075X09BTK4","interactive":"true","name":"COWSAY"}
 dagger --progress=plain \
     call \
-        -m github.com/shykes/daggerverse/wolfi@v0.1.2 \
+        -m github.com/shykes/daggerverse/wolfi@v0.1.4 \
```

Comment:
The previous version is crashing for me.

Reply:
makes sense, i think dagger releases sometimes break old versions 👍